tests/on_target/tests/conftest.py

```diff
@@ -5,24 +5,59 @@
 
 import os
 import re
-import pytest
 import types
+import pytest
+import subprocess
+from pytest_metadata.plugin import metadata_key
 from utils.flash_tools import recover_device
 from utils.uart import Uart, UartBinary
 from utils.hellonrfcloud_fota import HelloNrfCloudFOTA
-import sys
-sys.path.append(os.getcwd())
 from utils.logger import get_logger
 
 logger = get_logger()
 
 UART_TIMEOUT = 60 * 30
 
 SEGGER = os.getenv('SEGGER')
-UART_ID = os.getenv('UART_ID', SEGGER)
+UART_ID = SEGGER
```

Comment:
this is breaking connectivity bridge and dfu test, since now those test use env UART_ID_DUT_2 instead of UART_ID. Then get_uarts() wont be working for these two tests.